apps/s_server.c

```diff
@@ -2202,22 +2202,21 @@ static int sv_body(int s, int stype, int prot, unsigned char *context)
             BIO_printf(bio_err, "Turned on non blocking io\n");
     }
 
-    if (con == NULL) {
-        con = SSL_new(ctx);
+    con = SSL_new(ctx);
```

Comment:
Please add a check for NULL return; probably just as simple as
```
if (con == NULL)
    goto err;
```

Reply:
@richsalz Thank you for your review and comment.
I add error check for SSL_new, then I sent new pull request(#4560).

Reply:
@richsalz I canceled #4560, because I perceived that the error handling was not sufficient.
I'm going to send new request later.

Reply:
You know you can just add (or replace) patches in an existing pull request? Just add them to your branch and push that branch to github.

Reply:
@kroeckx Thank you for your information. Add new code my same branch.

@richsalz I add a check for NULL return. I found same problem in s_client.c and fixed it.